setup.py

```diff
@@ -23,6 +23,7 @@
     ],
     packages=[
         'geopyspark',
+        'geopyspark.command',
```

Comment:
I think that this branch might have to be rebassed, because I'm not sure why else this addition would be here. Also, `geopyspark.rasterio` needs to be added to this list.